assets/sass/scaffolding/_content.scss

```diff
@@ -90,6 +90,11 @@
         @include mq('medium-major') {
             flex: 0 0 320px;
         }
+
+        // Prevent sidebar from stretching to match primary content height
+        &.content-sidebar__secondary--minimal {
```

Comment:
Without this it looks... wrong:

![image](https://user-images.githubusercontent.com/394376/49936392-c9aa5680-fecb-11e8-9a84-7cc86fcee5b0.png)

Reply:
Rather than doing this could we take the manual tone classes off `content-sidebar__secondary` and make that grant summary thing use the card classes (or basically move the classes down a level). Should do the same thing without one-off styles.

Reply:
Done!